src/Http/SetRequestIpMiddleware.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry\Http;
+
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Sentry\State\HubInterface;
+use Sentry\State\Scope;
+
+/**
+ * This middleware enriches the Sentry scope with the IP address of the request.
+ * We do this ourself instead of letting the PHP SDK handle this because we want
+ * the IP from the Psr request because it takes into account trusted proxies.
```

Comment:
```suggestion
 * This middleware extends Sentry scope with IP address of the request. Because PSR request is used, it takes trusted 
 * proxies into account, which does not happen during automatic handling by PHP SDK.
```